PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] The commit message follows our guidelines: CONTRIBUTING.md#commit
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
```

Comment:
Wrap in triple backticks and remove dash (`-`) prefix like the *PR Type* section below. Then it doesn't show up as for example 1/3 tasks completed in various overviews.